Exercise1/myFile-rt/src/com/tridiumuniversity/myFile/BTestFile.java

```diff
@@ -0,0 +1,213 @@
+   */
+  public static final Property size = newProperty(0, ((BInteger)(BInteger.DEFAULT)).getInt(), null);
+  
+  /**
+   * Get the {@code size} property.
+   * @see #size
+   */
+  public int getSize() { return getInt(size); }
+  
+  /**
+   * Set the {@code size} property.
+   * @see #size
+   */
+  public void setSize(int v) { setInt(size, v, null); }
+
+////////////////////////////////////////////////////////////////
+// Property "createdDate"
+////////////////////////////////////////////////////////////////
+  
+  /**
+   * Slot for the {@code createdDate} property.
+   * @see #getCreatedDate
+   * @see #setCreatedDate
+   */
+  public static final Property createdDate = newProperty(0, BAbsTime.DEFAULT, null);
+  
+  /**
+   * Get the {@code createdDate} property.
+   * @see #createdDate
+   */
+  public BAbsTime getCreatedDate() { return (BAbsTime)get(createdDate); }
+  
+  /**
+   * Set the {@code createdDate} property.
+   * @see #createdDate
+   */
+  public void setCreatedDate(BAbsTime v) { set(createdDate, v, null); }
+
+////////////////////////////////////////////////////////////////
+// Property "modifiedDate"
+////////////////////////////////////////////////////////////////
+  
+  /**
+   * Slot for the {@code modifiedDate} property.
+   * @see #getModifiedDate
+   * @see #setModifiedDate
+   */
+  public static final Property modifiedDate = newProperty(0, BAbsTime.DEFAULT, null);
+  
+  /**
+   * Get the {@code modifiedDate} property.
+   * @see #modifiedDate
+   */
+  public BAbsTime getModifiedDate() { return (BAbsTime)get(modifiedDate); }
+  
+  /**
+   * Set the {@code modifiedDate} property.
+   * @see #modifiedDate
+   */
+  public void setModifiedDate(BAbsTime v) { set(modifiedDate, v, null); }
+
+////////////////////////////////////////////////////////////////
+// Action "touch"
+////////////////////////////////////////////////////////////////
+  
+  /**
+   * Slot for the {@code touch} action.
+   * @see #touch()
+   */
+  public static final Action touch = newAction(0, null);
+  
+  /**
+   * Invoke the {@code touch} action.
+   * @see #touch
+   */
+  public void touch() { invoke(touch, null, null); }
+
+////////////////////////////////////////////////////////////////
+// Action "print"
+////////////////////////////////////////////////////////////////
+  
+  /**
+   * Slot for the {@code print} action.
+   * @see #print()
+   */
+  public static final Action print = newAction(0, null);
+  
+  /**
+   * Invoke the {@code print} action.
+   * @see #print
+   */
+  public void print() { invoke(print, null, null); }
+
+////////////////////////////////////////////////////////////////
+// Type
+////////////////////////////////////////////////////////////////
+  
+  @Override
+  public Type getType() { return TYPE; }
+  public static final Type TYPE = Sys.loadType(BTestFile.class);
+
+/*+ ------------ END BAJA AUTO GENERATED CODE -------------- +*/
+public void doTouch()
+{
+    System.out.println("Hello, World  Touch");
```

Comment:
This is fine to include while you're testing your code, but it's best to remove this once you're done testing. Otherwise, you'll have `Hello, World Touch` printed to stdout every time you invoke this action! That'd be unexpected behavior for a user.